controllers/content/ContentController.php

```diff
@@ -18,7 +18,7 @@
 class ContentController extends BaseController
 {
 
-    public function actionFindByContainer($id)
+    public function actionFindByContainer($id, $order_by = 'created_at', $date_updated_from = null, $date_updated_to = null)
```

Comment:
- Please make sure that we always use `camelCase` variables.

- Instead of accepting a free database column, it would probably be more reliable to just accept two options (as in the regular stream) "creationTime" (default) and "lastUpdate". 

I would also have some concerns about SQL injectoins with your current solution.